nimpretty/nimpretty.nim

```diff
@@ -25,11 +25,11 @@ const
 Usage:
   nimpretty [options] file.nim
 Options:
-  --backup:on|off     create a backup file before overwritting (default: ON)
-  --output:file       set the output file (default: overwrite the .nim file)
-  --indent:N          set the number of spaces that is used for indentation
-  --version           show the version
-  --help              show this help
+  --backup:on|off[=on]  when on, create a backup file if input is overwritten
+  --output:file[=input] set the output file
```

Comment:
``--output:file=input``? That's more confusing than helpful.

Reply:
changed back to `--output:file         set the output file (default: overwrite the input file)`
(removed .nim from original code since it can be used with .nims)